content/es/docs/topics/provenance.md

```diff
@@ -0,0 +1,292 @@
+=aHfz
+-----END PGP SIGNATURE-----
+```
+
+Tenga en cuenta que la sección YAML contiene dos documentos (separados por `...\n`).
+El primer archivo es el contenido de `Chart.yaml`. El segundo son las sumas de
+comprobación, un mapa de nombres de archivo a resúmenes SHA-256 del contenido de
+ese archivo en el momento del empaquetado.
+
+El bloque de firma es una firma PGP estándar, que proporciona [resistencia a la
+manipulación(https://www.rossde.com/PGP/pgp_signatures.html).
+
+## Repositorios de Charts
+
+Los repositorios de charts sirven como una colección centralizada de charts de Helm.
+
+Los repositorios de charts deben permitir la entrega de archivos de procedencia
+a través de HTTP a través de una solicitud específica, y deben hacerlos disponibles
+en la misma ruta de URI que el chart.
+
+Por ejemplo, si la URL base de un paquete es
+`https://example.com/charts/mychart-1.2.3.tgz`, el archivo de procedencia,
+si existe, DEBE ser accesible en `https://example.com/charts/mychart-1.2.3.tgz.prov`.
+
+Desde la perspectiva del usuario final, `helm install --verify myrepo/mychart-1.2.3`
+debería resultar en la descarga tanto del chart como del archivo de procedencia
+sin configuración o acción adicional del usuario.
+
+## Establecimiento de Autoridad y Autenticidad
+
+Cuando se trata de sistemas de cadena de confianza, es importante poder establecer
+la autoridad de un firmante. O, para decirlo claramente, el sistema anterior depende
+del hecho de que usted confía en la persona que firmó el chart. Eso, a su vez,
+significa que debe confiar en la clave pública del firmante.
+
+Una de las decisiones de diseño de Helm ha sido que el proyecto Helm no se
+insertará en la cadena de confianza como parte necesaria. No queremos ser
+"la autoridad de certificación" para todos los firmantes de charts. En cambio,
+estamos a favor de un modelo descentralizado, que es parte de la razón por la
+que elegimos OpenPGP como nuestra tecnología fundamental. Entonces, cuando se
+trata de establecer la autoridad, hemos dejado este paso más o menos indefinido
+en Helm 2 (una decisión llevada a cabo en Helm 3).
+
+Sin embargo, tenemos algunos consejos y recomendaciones para aquellos interesados
+en utilizar el sistema de procedencia:
+
+- La plataforma [Keybase](https://keybase.io) proporciona un repositorio público
+  centralizado para información de confianza.
+  - Puede utilizar Keybase para almacenar sus claves o para obtener las claves
+    públicas de otros.
+  - Keybase also has fabulous documentation available
+  - While we haven't tested it, Keybase's "secure website" feature could be used
+    to serve Helm charts.
+  - The basic idea is that an official "chart reviewer" signs charts with her or
+    his key, and the resulting provenance file is then uploaded to the chart
+    repository.
+  - There has been some work on the idea that a list of valid signing keys may
+    be included in the `index.yaml` file of a repository.
```

Comment:
Did you intend to keep some English in this doc? Thanks!

Reply:
No, I think I missed those lines.